### ISSUE TITLE
up to date

### ISSUE THREAD

**Issue by fakis2232** [iRebbok/RemotePermissionCard#8]
![MultiAdmin_WoJ5NqO1xj](https://user-images.githubusercontent.com/78535464/106874837-db789480-66f7-11eb-95d6-79d25acb1e49.png)
what to do

**Comment by iRebbok** [comment]
You're probably trying using a build for EXILED with SMod. There's one release for SMod: https://github.com/iRebbok/RemoteKeycard/releases/tag/1.4.1-3.6.0. I stopped maintaining the branch for SMod, if the SMod devs break something, I won't be updating the plugin.

**Comment by fakis2232** [comment]
well, when I run the plugin on 1.4.1-3.6.0, it does not work for me, the doors do not open, but it is written in the console that it is loaded

**Comment by iRebbok** [comment]
I don't know, maybe it requires an update or it's an issue from your side.

**Comment by fakis2232** [comment]
maybe but scp Smod i reinstall and not helped

**Comment by fakis2232** [comment]
but it used to work

**Comment by fakis2232** [comment]
Maybe you can try to get me a plug in please

**Comment by fakis2232** [comment]
plugin*

**Comment by fakis2232** [comment]
can you try running this plugin please?

**Comment by iRebbok** [comment]
No, as I said, stopped supporting SMod. There's a branch for SMod since the release, if you want to hire a dev to get it working.
When I said update I meant an update for the plugin, not SMod itself.

**Comment by fakis2232** [comment]
is this how it should be?
![MultiAdmin_9gHyDYNfux](https://user-images.githubusercontent.com/78535464/107206125-75a24a80-6a20-11eb-990b-f2eb4c3bb635.png)

**Comment by mixonNOK1A** [comment]
hey , Im using only exiled platform , what should I put in the plugins folder from https://github.com/iRebbok/RemoteKeycard/releases/tag/1.7.0-EXILED2.9.0 , and how to properly set it in the configs (what exactly should I write in cofigs)? please help
